### PR TITLE
[Refactoring] Remove unused methods from protocols maven modules

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: protocols/netty/src/main/java/org/apache/james/protocols/netty/ConnectionPerIpLimitUpstreamHandler.java [hunk 1]
```diff
@@ -46,19 +46,6 @@ public ConnectionPerIpLimitUpstreamHandler(int maxConnectionsPerIp) {
         this.maxConnectionsPerIp = maxConnectionsPerIp;
```
Comment (remk) [inline protocols/netty/src/main/java/org/apache/james/protocols/netty/ConnectionPerIpLimitUpstreamHandler.java]: `maxConnectionsPerIp` could now be declared final and the `volatil` keyword be removed

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
```
[ERROR] /james-project/server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/OioIMAPServer.java:[46,5] method does not override or implement a method from a supertype
```

**Comment by chibenwa** [comment]
`DefaultMailboxesProvisioningFilterTest.createMailboxesIfNeededShouldNotGenerateExceptionsInConcurrentEnvironment:94 » Execution`

test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**